sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/expressions/regexpExpressions.scala

```diff
@@ -229,36 +229,58 @@ case class RLike(left: Expression, right: Expression) extends StringRegexExpress
 
 
 /**
- * Splits str around pat (pattern is a regular expression).
+ * Splits str around matches of the given regex.
  */
 @ExpressionDescription(
-  usage = "_FUNC_(str, regex) - Splits `str` around occurrences that match `regex`.",
+  usage = "_FUNC_(str, regex, limit) - Splits `str` around occurrences that match `regex`" +
+    " and returns an array of at most `limit`",
+  arguments = """
+    Arguments:
+      * str - a string expression to split.
+      * regex - a string representing a regular expression. The regex string should be a
+        Java regular expression.
+      * limit - an integer expression which controls the number of times the regex is applied.
+
+        limit > 0: The resulting array's length will not be more than `limit`,
+                   and the resulting array's last entry will contain all input
+                   beyond the last matched regex.
+        limit <= 0: `regex` will be applied as many times as possible, and
+                    the resulting array can be of any size.
```

Comment:
I would do this:

```
        * limit > 0: The resulting array's length will not be more than `limit`,
           and the resulting array's last entry will contain all input
           beyond the last matched regex.
        * limit <= 0: `regex` will be applied as many times as possible, and
           the resulting array can be of any size.
```

This is just markdown rendered by mkdocs; so probably, it's better to make it consistent with other docs.

Reply:
Current bullt doc is:

![screen shot 2018-09-03 at 10 04 46 am](https://user-images.githubusercontent.com/6477701/44964013-e47e3e00-af60-11e8-975a-41d5dc5f1e6e.png)

which doesn;t look quite nice. Please verify the output by referring https://github.com/apache/spark/tree/master/docs

Reply:
@HyukjinKwon sorry, but where can I get the markdown output of the `regexExpressions.scala`? When I run unidoc, I'm only seeing the html corresponding to python, java, r (and these are from the functions, not from `regexExpressions.scala`.

I'm running 
```
build/sbt unidoc
```

Reply:
got it, nvm